linkerd/docs/namer.md

```diff
@@ -610,18 +611,19 @@ prefix | `io.l5d.dnssrv` | Resolves names with `/#/<prefix>`.
 experimental | `false` | Since the DNS-SRV namer is still considered experimental, this must be set to `true`.
 refreshIntervalSeconds | `5` | linkerd will perform a SRV lookup for each host every `refreshIntervalSeconds`.
 dnsHosts | `<empty list>` | If specified, linkerd will use these DNS servers to perform SRV lookups. If not specified, linkerd will use the default system resolver.
- 
+
 ### DNS-SRV Path Parameters
 
 > Dtab Path Format
 
-```
+```yaml
 /#/<prefix>/<address>
```

Comment:
Did you know? Having a code block without a format param is usually ok, but in this case totally breaks because of the `#` sign in side the block. So we're using `yaml` here and elsewhere in the document, even though the contents aren't yaml.